server/services/query-memory-db/queries/notify-business-process-assigned.js

```diff
@@ -4,8 +4,9 @@ var ensureDatabase = require('dbjs/valid-dbjs')
   , notifyAssigned = require('../../../email-notifications/business-process-assigned');
 
 module.exports = function (db) {
-	var dispatcherEmails = ensureDatabase(db).User.filterByKey('roles', function (roles) {
+	var dispatcherEmails = ensureDatabase(db).User.filterByKey('roles', function (roles, user) {
 		if (!roles) return false;
+		if (user.isSuperUser) return false;
```

Comment:
This won't work as change of `isSuperUser` will not update collection.

It has to be done possibly as union of two collections: `dispatchers.and(notSuperUsers)`